Studio-Lovelies/GG-JointJustice-Unity
# Feature/Flashing witness testimony sign and cross examination animation

## Summary

Adds the flashing witness testimony sign, as well as a command to enable and disable it.
Also adds the sprites and animation for the cross examination animation intro.
These are demonstrated in the WitnessTestimonyTest Ink script, which plays the animations using Ink knots provided in a Macros script. This allows the correct actions to be run when starting a cross examination or witness testimony.

## Before/after screenshots and/or animated gif

![witnesstestimony](https://user-images.githubusercontent.com/67964179/148698196-dcbedeae-0d89-436a-a761-84399b43acdb.gif)

## Testing instructions

Play WitnessTestimonyTest Ink script.
Run the tests and confirm they pass (make sure WitnessTestimonyTest is not the current Ink script in Inky-TestScene because beginning a scene with an animation breaks some tests)

## Additional information

- `[x]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  
closes #134
closes #106
